docs/Contributing.asciidoc

```diff
@@ -226,7 +226,9 @@ Assuming you have already followed steps 1. to 4. above:
 1. Create a separate database: `createdb -O your_username openqa-o3+ where `openqa-o3+ is
    the name you want to use for the database
 2. The next steps must be done by the user you start your local openQA instance with.
-3. Import dump: `pg_restore -c -d openqa path/to/dump`
+3. Import dump: `pg_restore -c -d openqa-o3+ path/to/dump`
+   Note that errors of the form `ERROR:  role "geekotest" does not exist` are due to the users
+   in the production setup and can safely be ignored. Everything will be owned by `your_username`.
```

Comment:
Seems like your editor forgot to wrap lines :)

Reply:
I went by the precedent of existing lines

Reply:
well then, care to fix it?